allennlp/data/dataset_readers/universal_dependencies.py

```diff
@@ -59,7 +62,10 @@ def _read(self, file_path: str):
                 heads = [x["head"] for x in annotation]
                 tags = [x["deprel"] for x in annotation]
                 words = [x["form"] for x in annotation]
-                pos_tags = [x["upostag"] for x in annotation]
+                if self.use_language_specific_pos:
+                    pos_tags = [x["xpostag"] for x in annotation]
```

Comment:
Would this problem have been caught earlier if you had used `pos_tags` instead of `pos` for your label namespace in the pos tag sequence label field?  I think it would have, because you'd have crashed at an UNK part of speech tag.  I think that's the right thing to do, as this is a closed class that you can expect to see all of at training time (and if not, you can manually add all of them to the vocabulary).